apps/meteor/client/views/admin/settings/Setting/Setting.tsx

```diff
@@ -105,12 +106,21 @@ function Setting({ className = undefined, settingId, sectionChanged }: SettingPr
 			) : undefined,
 		[i18n, i18nDescription, t],
 	);
+
 	const callout = useMemo(
 		() => alert && <span dangerouslySetInnerHTML={{ __html: i18n.exists(alert) ? t(alert) : alert }} />,
 		[alert, i18n, t],
 	);
 
-	const shouldDisableEnterprise = setting.enterprise && !isEnterprise;
+	const hasSettingModule = useMemo(() => {
+		if (!setting?.modules) {
+			return false;
+		}
+
+		return setting.modules.every((module) => activeModules.includes(module as LicenseModule));
```

Comment:
Is it possible for `setting.modules` to be an empty array? 
If yes, `every()` of an empty array is always `true`, could this be a problem? 🤔